docs/babel.config.js

```diff
@@ -33,17 +14,7 @@ module.exports = function getBabelConfig(api) {
       // backport of https://github.com/zeit/next.js/pull/9511
       ['next/babel', { 'transform-runtime': { corejs: 2, version: transformRuntimeVersion } }],
     ],
-    plugins: [
-      ...baseConfig.plugins,
-      'babel-plugin-preval',
-      [
-        'babel-plugin-module-resolver',
```

Comment:
Maybe we can remove this package now if it is no longer used? 🤔

Reply:
I think it's still used by some of the tests